psydac/api/feec.py

```diff
@@ -86,51 +86,88 @@ def __init__(self, mapping, *spaces):
     #--------------------------------------------------------------------------
     @property
     def dim(self):
+        """dimension of the physical and logical space"""
         return self._dim
 
     @property
     def V0(self):
+        """First space of the De Rham sequence : H1 space"""
         return self._spaces[0]
 
     @property
     def V1(self):
+        """Second space of the De Rham sequence :
+        1d : L2 space
+        2d : either Hdiv or Hcurl space
+        3d : Hcurl space"""
         return self._spaces[1]
 
     @property
     def V2(self):
+        """Third space of the De Rham sequence :
+        2d : L2 space
+        3d : Hdiv space"""
         return self._spaces[2]
 
     @property
     def V3(self):
+        """Fourth space of the De Rham sequence : L2 space in 3d"""
         return self._spaces[3]
 
     @property
     def H1vec(self):
+        """Vectorial H1 space built has cartesian product of V0"""
```

Comment:
typo: build _as_

Reply:
also please specify: `... cartesian product of V0 n times, with n = dimension of (logical) domain`

Reply:
I would also replace "Vectorial" with "Vector", which is much more common in English.